changelogs/unreleased/gh-8051-set-box-cfg-thru-env.md

```diff
@@ -0,0 +1,4 @@
+## feature/box/cfg
```

Comment:
It seems that you added 00000000000000000000.snap by mistake.

Reply:
Oh, fixed.